packages/fiori/src/UserSettingsAppearanceView.ts

```diff
@@ -0,0 +1,114 @@
+import UserSettingsView from "./UserSettingsView.js";
+import UserSettingsAppearanceViewTemplate from "./UserSettingsAppearanceViewTemplate.js";
+import UserSettingViewCss from "./generated/themes/UserSettingsView.css.js";
+import type UserSettingsAppearanceViewItem from "./UserSettingsAppearanceViewItem.js";
+import type UserSettingsAppearanceViewGroup from "./UserSettingsAppearanceViewGroup.js";
+
+import {
+	customElement, slot, eventStrict as event, property,
+} from "@ui5/webcomponents-base/dist/decorators.js";
+import jsxRenderer from "@ui5/webcomponents-base/dist/renderer/JsxRenderer.js";
+
+@customElement({
+	tag: "ui5-user-settings-appearance-view",
+	renderer: jsxRenderer,
+	template: UserSettingsAppearanceViewTemplate,
+	styles: [UserSettingViewCss],
+})
+
+/**
+ * Fired when a theme is selected.
+ * @public
+ */
+@event("theme-selected")
+
+/**
+ * @class
+ * ### Overview
+ *
+ * The `ui5-user-settings-appearance-view` represents a view displayed in the `ui5-user-settings-item`.
+ *
+ * ### ES6 Module Import
+ * `import "@ui5/webcomponents-fiori/dist/UserSettingsAppearanceView.js";`
+ *
+ * @constructor
+ * @extends UserSettingsView
+ * @experimental
+ * @public
+ * @since 2.17.0
+ */
+class UserSettingsAppearanceView extends UserSettingsView {
+	eventDetails!: {
+		"theme-selected": { selectedTheme: string };
+	}
+
+	/**
+	 * Defines the currently selected theme key.
+	 * @default ""
+	 * @public
+	 */
+	@property()
+	selectedItemKey = "";
```

Comment:
its better to remove this property and use the selected property of the appearance items.

Check for reference UserSettingsDialog and UserSettingsItem - the same logic applies there